connect-inject/health_check_resource_test.go

```diff
@@ -566,7 +566,7 @@ func TestReconcile_IgnoreStatuses(t *testing.T) {
 					Ready: false,
 				},
 				{
-					Name: "consul-connect-lifecycle-sidecar",
+					Name: "consul-connect-consul-sidecar",
```

Comment:
Should we call this `consul-sidecar` or `consul-connect-sidecar`?

Reply:
I think I still like `consul-connect-consul-sidecar` because the component and command is consul-sidecar and the consul-connect is a prefix for other containers like the envoy sidecar name to specify its only added with Connect. `consul-sidecar` alone doesn't let you know this is a Connect-related feature, and loses consistency with the envoy container name and `consul-connect-sidecar` also loses consistency with the envoy container name. Wdyt?

Reply:
I think I'm with Luke on this one. I made a similar comment in another place in the code before I noticed it was being discussed here. Personally, i wouldn't mind renaming `consul-connect-envoy-sider` to `envoy-sidecar` and the calling the sidecar in question the `consul-sidecar`. 
I like the argument for consistency but in this case, IMO, it feels more confusing to parse.

Reply:
I've renamed the containers to `consul-sidecar` and `envoy sidecar` :)